patches/0013-remove-long-path-support-hack.patch

```diff
@@ -0,0 +1,112 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: qmuntal <qmuntaldiaz@microsoft.com>
+Date: Wed, 3 Apr 2024 12:38:51 +0200
+Subject: [PATCH] remove long path support hack
+
+Upstream Go tricks Windows into enabling long path support by setting an
+undocumented flag in the PEB. The Microsoft Go fork can't use undocumented
+APIs, so this commit removes the hack.
+
+There is no documented way to enable long path support from within the
+process, so this this is a breaking change for the Microsoft Go fork.
+Note that the Go standard library makes a best effort to support long
+paths by using the `\\?\` prefix when possible, so this change should
+only affect long relative paths, which can't be used with the `\\?\`.
+
```

Comment:
Doesn't round trip for me (happened to notice when checking out https://github.com/microsoft/go/pull/1182#discussion_r1550526290)

```suggestion
```